cpp/modmesh/modmesh.hpp

```diff
@@ -40,4 +40,11 @@
 #include <modmesh/mesh/mesh.hpp>
 #include <modmesh/toggle/toggle.hpp>
 
+// TODO Add MSVC case once sanitizer can be default turned on for CI testing
+#if defined(USE_SANITIZER) && (defined(__clang__) || defined(__GNUC__))
+#define ATTRIBUTE_NO_SANITIZE_ADDRESS __attribute__((no_sanitize_address))
```

Comment:
What about we name this kind of macros as `ASAN_...`?

Reply:
Thanks for the recommendations, I have change the marco in https://github.com/solvcon/modmesh/pull/261/commits/58ed2df0460ce2ed2230d09a6cc411a0461d373a